apiserver/handler.go

```diff
@@ -0,0 +1,230 @@
+package apiserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"istio.io/manager/model"
+
+	restful "github.com/emicklei/go-restful"
+	"github.com/golang/glog"
+)
+
+var schema model.ProtoSchema
```

Comment:
(see https://github.com/istio/manager/pull/446/files#diff-d6311766e3a7c1d9998905f0760eccdbR79 where we had this exact problem)